src/components/security_manager/test/ssl_context_test.cc

```diff
@@ -158,8 +158,8 @@ class SSLTest : public testing::Test {
     client_ctx = client_manager_->CreateSSLContext();
 
     security_manager::SSLContext::HandshakeContext ctx;
-    ctx.make_context(custom_str::CustomString("SPT"),
-                     custom_str::CustomString("client"));
+    ctx.expected_cn = "SPT";
+    ctx.expected_sn = "client";
     server_ctx->SetHandshakeContext(ctx);
```

Comment:
@nk0leg  in previous version it was one object< i do not want to add some unexpected regression